plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/RetryingConnectionFactory.java

```diff
@@ -14,6 +14,8 @@
 package io.trino.plugin.jdbc;
```

Comment:
Please explain what value provides this refactor (what is the motivation behind this change).

Reply:
Please take a look at improved commit messages